lib/std/fs/file.zig

```diff
@@ -385,6 +385,274 @@ pub const File = struct {
         try os.fchown(self.handle, owner, group);
     }
 
+    /// Cross-platform representation of permissions on a file.
+    /// The `readonly` and `setReadonly` are the only methods available across all platforms.
+    /// Unix-only functionality is available through the `unixHas` and `unixSet` methods.
+    pub const Permissions = switch (builtin.os.tag) {
+        .windows => struct {
+            attributes: os.windows.DWORD,
+
+            const Self = @This();
+
+            /// Returns `true` if permissions represent an unwritable file.
+            /// On Unix, `true` is returned only if no class has write permissions.
+            pub fn readonly(self: Self) bool {
```

Comment:
Nitpick: I think these functions would be better capitalized as `readOnly` / `setReadOnly`

Reply:
I agree